src/JobApp.re

```diff
@@ -2,31 +2,35 @@ type state = {
   url: string,
   company: string,
   position: string,
+  postedDate: string,
 };
 
 type action =
   | UpdateUrl(string)
   | UpdateCompany(string)
-  | UpdatePosition(string);
+  | UpdatePosition(string)
+  | UpdatePostedDate(string);
```

Comment:
not that this matters too much, but is there any reason why you didn't use 

https://bucklescript.github.io/bucklescript/api/Js_date.html?

Reply:
> yeah if possible, if it's not too much bother if it takes more than 5 mins then forget it

Yep, it only takes a `Ctrl-H` to replace `Js.Date` with `Js_date`.
But isn't the current notation more homogenous with the rest of the `Js` API we are using: 
`Js.Log`, `Js.Re`,  `Js.String`, `Js.Option`, `Js.Exn`

Reply:
well I meant it more as in why is the type of postedDate a string and not of date type?

Reply:
ahhh completely misunderstood your point then.

It is because the `postedDate` state is used to set the `<input type='date'>` placeholder.
This placeholder needs to be a string in the format `yyyy-MM-dd`.

Storing the date in `Js.Date.t` type forces me to transform the value every time the state changes to the appropriate string format.

Reply:
all gucci then